aws/dev.yml

```diff
@@ -47,28 +47,6 @@
 
     - debug: var=stack
 
-    - debug: msg={{ stack | json_query('stack_resources[?logical_resource_id==`FxaELB`].physical_resource_id | [0]') }}
-
-    - name: Waiting for instance to join ELB (ignore FAILED RETRYING log messages)
```

Comment:
This check obviously wasn't working after I switched away from the ELB, if we want to keep the ALB then I guess we'll need to find an equivalent.